src/azure-cli/azure/cli/command_modules/serviceconnector/_validators.py

```diff
@@ -286,18 +286,18 @@ def get_local_context_value(cmd, arg):
 def opt_out_auth(namespace):
     '''Validate if config and auth are both opted out
     '''
-    opt_out_list = namespace.opt_out_list
```

Comment:
Can we just change the line `opt_out_list = getattr(namespace, 'opt_out_list', None)`